dashboard/callbacks.py

```diff
@@ -27,11 +27,12 @@
 from utilities import _create_html
 import dianna
 from dianna.utils.tokenizers import SpacyTokenizer
+from pathlib import Path
 
 
 warnings.filterwarnings('ignore')  # disable warnings relateds to tf versions
 
-FOLDER_ON_SERVER = "app_data"
+FOLDER_ON_SERVER = str(Path(__file__).parent.absolute()) + "/app_data"
```

Comment:
I just realized this would be the intended way to do path concatenation using `pathlib`:

```suggestion
FOLDER_ON_SERVER = str(Path(__file__).parent.absolute() / "app_data")
```